nautobot/dcim/tables/devicetypes.py

```diff
@@ -61,6 +63,28 @@ class Meta(BaseTable.Meta):
         )
 
 
+#
+# Hardware Family
+#
+
+
+class HardwareFamilyTable(BaseTable):
+    pk = ToggleColumn()
+    name = tables.LinkColumn()
```

Comment:
`LinkColumn` is deprecated

```suggestion
    name = tables.Column(linkify=True)
```